chef/cookbooks/haproxy/resources/loadbalancer.rb

```
@@ -27,4 +27,5 @@
 attribute :mode,    kind_of: String,  default: "http", equal_to: ["http", "tcp", "health"]
 attribute :use_ssl, kind_of: [TrueClass, FalseClass], default: false
 attribute :options, kind_of: Array,   default: []
+attribute :check,   kind_of: Hash,    default: { inter: 2000, rise: 2, fall: 5 }
```

I think we should be more aggressive when doing health checks: faster checks and failing faster. At least I would use the same as the http case:
`inter 1000 downinter 3000 rise 3 fall 1`. I would even set `inter` to `250`, to do four checks per second.